_contrib/governance.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Governance
+layout: guide
+---
+
+## Making decisions
+
+There's two main pieces to the way decisions are made in hyper:
+
+1. A decision-making framework
+2. The people who apply it
+
+The people are described [lower down in this document](#roles).
+
+### Decision-making framework
+
+We start with the users. The project wouldn't exist without them, and it exists
+in order to enable users to do amazing things with HTTP. We listen to our
+users. Some actively contribute their thoughts, but many others we must seek
+out to learn about their usage, joys, and headaches. Those insights allow our
+experts to determine the best solutions for the users.
+
+We then define a set of [tenets](./tenets.md), which are guiding principles
```

Comment:
I opted to also changes the names of the links to lowercase, but if you'd like to keep them capitalized (`[TENETS](./tenets.md)`) I'm happy to change it back (which will also reduce the CI complexity).

Reply:
Seems fine being lowercased. Or uppercased, if that's easier. Either, really :)

Reply:
Cool, I'll leave it like this then for now.